lib/icalendar/util/deserialize.ex

```diff
@@ -21,8 +21,18 @@ defmodule ICalendar.Util.Deserialize do
   """
   def retrieve_kvs(line) do
     # Split Line up into key and value
-    [key, value] = String.split(line, ":", parts: 2, trim: true)
-    [key, params] = retrieve_params(key)
+    {key, value, params} =
+      case String.split(line, ":", parts: 2, trim: true) do
+        [key, value] ->
+          [key, params] = retrieve_params(key)
+          {key, value, params}
+
+        [key] ->
+          {key, nil, %{}}
+      end
+
+    # [key, value] = String.split(line, ":", parts: 2, trim: true)
+    # [key, params] = retrieve_params(key)
```

Comment:
Could you remove these comments please?

Reply:
https://github.com/lpil/icalendar/pull/36/commits/61889b7b29baee67c8e8c661f23837f1165e82ba